src/cogent3/app/__init__.py

```diff
@@ -33,24 +33,23 @@ def _get_app_attr(name, is_composable):
         name,
         is_composable,
         obj.__doc__,
-        _types["_data_types"],
-        _types["_return_types"],
+        ", ".join(sorted(_types["_data_types"])),
+        ", ".join(sorted(_types["_return_types"])),
```

Comment:
`sorted` changes order, but we may need order later.

Reply:
We may need order to find type of the first argument of `main` function.

Reply:
Not in the table. Note, the original app attributes are `frozenset()` (thus immutable).

These lists are created only to produce this table.

Reply:
OK, Now I understand it is only for this table.